packages/bootstrap-vue-next/src/plugins/createBootstrap/index.ts

```
@@ -1,40 +1,32 @@
 // Main app plugin
 export const createBootstrap = (pluginData: Readonly<BootstrapVueOptions> = {}): Plugin => ({
   install(app) {
-    if (pluginData.breadcrumb ?? true === true) {
-      app.use(breadcrumbPlugin)
-    }
-    if (pluginData.modalController ?? true === true) {
-      app.use(modalControllerPlugin)
-    }
-    if (pluginData.modalManager ?? true === true) {
-      app.use(modalManagerPlugin)
+    if (pluginData.registries ?? true === true) {
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)